rules-tests/CodingStyle/Rector/Namespace_/ImportFullyQualifiedNamesRector/AutoImportNamesOnlyOnChangedFiles.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\CodingStyle\Rector\Namespace_\ImportFullyQualifiedNamesRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+/**
+ * @see \Rector\PostRector\Rector\NameImportingPostRector
+ */
+final class AutoImportNamesOnlyOnChangedFiles extends AbstractRectorTestCase
```

Comment:
should has Test suffix to make it runnable from directory:

```suggestion
final class AutoImportNamesOnlyOnChangedFilesTest extends AbstractRectorTestCase
```

Reply:
How could i forgot.